### ISSUE TITLE
Put items contents into JSON

### ISSUE THREAD

**Issue by ddenisenko** [raml-org/raml-js-parser-2#468]
Copy-pasting from https://github.com/raml-org/raml-js-parser-2/issues/291#issuecomment-248605886

I'm not sure if this is related, but it looks like expanded types don't recursively expand.

Example:

```
types:
  MyType:
    properties:
      id:
        type: number
      boundaries?:
        type: object[]
        description: An array of boundaries
        items:
          properties:
            id:
              type: string
            type:
              type: string
            boundaryAsWkt:
              type: string
              description: The well-known-text value of the boundary.
```

Expands toJSON():

(snipped):

```
{ name: 'boundaries',
  displayName: 'boundaries',
  type: [ 'array' ],
  required: false,
  description: 'An array of boundaries',
  items: 'object',
  __METADATA__: { primitiveValuesMeta: { displayName: [Object], required: [Object] } } }
```

I cannot access the expected propertied under the JSON type (this is actually coming from raml2obj but using 1.1.2 parser and `expand(true)`).

**Comment by ashleydw** [comment]
Can someone point me to where I could potentially fix this?

**Comment by KonstantinSviridov** [comment]
@ashleydw the fix will be published in 1.1.4. For now you may apply the same fix as in https://github.com/raml-org/raml-js-parser-2/commit/745d7f48d296a6e246267694f35e8f6226af7395
